smart_ats/jobs/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from smart_ats.companies.models import *
+from taggit.managers import TaggableManager
+# Create your models here.
+
+class Tag(models.Model):
+    #name=models.CharField(max_length = 200,null=True)
+    name=TaggableManager();
+
+class Category(models.Model):
+    title =models.CharField(max_length = 200,null=True)
```

Comment:
it's not nullable.
we need db_index.
30 char will be enough.

it's required to track created and updated at dates and times.
it a good idea to use `from model_utils.models import TimeStampedModel`